rest/route/build_test.go

```diff
@@ -21,8 +22,8 @@ import (
 // Tests for fetch build by id
 
 type BuildByIdSuite struct {
-	sc   *data.MockConnector
-	data data.MockBuildConnector
+	sc   *data.DBConnector
+	data data.DBBuildConnector
```

Comment:
it's not really clear to me why we need both data and DBConnector; should we just get rid of data?